src/mumble/AudioInput.cpp

```diff
@@ -31,6 +31,14 @@ extern "C" {
 #include <exception>
 #include <limits>
 
+#ifdef USE_RNNOISE
+/// Clip the given float value to a range that can be safely converted into a short (without causing integer overflow)
+static short clampFloatSample(float v) {
+	return static_cast< short >(std::min(std::max(v, static_cast< float >(std::numeric_limits< short >::min())),
+										 static_cast< float >(std::numeric_limits< short >::max())));
+}
+#endif
```

Comment:
Now that we can make use of C++17: [`std::clamp`](https://en.cppreference.com/w/cpp/algorithm/clamp)